### DISCUSSION TITLE
文章中添加的图库，前台如何正确显示？

### DISCUSSION THREAD

**Discussion by aqyygao** [Tokinx/Wing#141] [问答] [answered]
文章编辑中添加的图库区块，在前台显示竖向排列，如何能正确显示？

**Comment by Tokinx** [comment]
现在显示有什么问题吗？

**Reply by aqyygao** [reply]
![1711362458017](https://github.com/Tokinx/Wing/assets/134024961/759ca0a0-481c-487b-ab26-95ebebe4219b)
![微信图片_20240325182828](https://github.com/Tokinx/Wing/assets/134024961/9205acc8-b7df-461b-bd6b-207406ff446d)

图库文章显示竖排排列，不知道是不是我的问题。

**Reply by Tokinx** [reply, marked as answer]
好的，wp更新了这个布局有点调整，稍等发个推送哈。

**Reply by aqyygao** [reply]
辛苦大佬👍
